content/contributing/first-time/what/contents.lr

```diff
@@ -53,22 +53,16 @@ may also find it helpful to filter the tickets for the platform you actually use
 for development - it will be difficult to fix a Linux bug if you don't have a
 Linux machine to use for testing!
 
-Pick a ticket, leave a comment that you're looking into the problem, and try
-to develop a pull request implementing a fix!
-
-Colosseum
-~~~~~~~~~
+If you would like to contribute to a specific project, you can also filter good
+first issues by specific projects using these links:
 
-If you like the idea of a very well specified, but complex problem, `Colosseum
-<https://github.com/beeware/colosseum>`__ is a good place to contribute.
-Colosseum is a pure-Python, browser implementation of the CSS layout algorithm.
-BeeWare uses it to lay out widgets on the screen of an application. The CSS
-specification has an extensive test suite; Colosseum needs to pass all those
-tests.
+- `BeeWare <https://github.com/search?q=repo%3Abeeware%2Fbeeware+label%3A%22good+first+issue%22+is%3Aissue+is%3Aopen&type=issues>`__
+- `beeware.github.io <https://github.com/search?q=repo%3Abeeware%2Fbeeware.github.io+label%3A%22good+first+issue%22+is%3Aissue+is%3Aopen&type=issues>`__
+- `Briefcase <https://github.com/search?q=repo%3Abeeware%2Fbriefcase+label%3A%22good+first+issue%22+is%3Aissue+is%3Aopen&type=issues>`__
+- `Toga <https://github.com/search?q=repo%3Abeeware%2Ftoga+label%3A%22good+first+issue%22+is%3Aissue+is%3Aopen&type=issues>`__
```

Comment:
Looks good, but I would reorder these so that (1) the "real" projects are on the list first, and (2) uses project names rather than the GitHub repos.
```suggestion
- `Briefcase <https://github.com/search?q=repo%3Abeeware%2Fbriefcase+label%3A%22good+first+issue%22+is%3Aissue+is%3Aopen&type=issues>`__
- `Toga <https://github.com/search?q=repo%3Abeeware%2Ftoga+label%3A%22good+first+issue%22+is%3Aissue+is%3Aopen&type=issues>`__
- `The BeeWare tutorial <https://github.com/search?q=repo%3Abeeware%2Fbeeware+label%3A%22good+first+issue%22+is%3Aissue+is%3Aopen&type=issues>`__
- `The BeeWare website <https://github.com/search?q=repo%3Abeeware%2Fbeeware.github.io+label%3A%22good+first+issue%22+is%3Aissue+is%3Aopen&type=issues>`__
```